read-the-docs/source/Bugzilla.rst

```diff
@@ -24,6 +25,17 @@ Methods
 -  `add_cc() <#add_cc>`__
 -  `add_attachment() <#add_attachment>`__
 
+get_ticket_content()
+--------------------
+
+``get_ticket_content(self, ticket_id=None)``
+
+Queries the Bugzilla API to get ticket_content using ticket_id.
+
+.. code:: python
+
+    t = ticket.create(ticket_id='Ticket_ID')
+
```

Comment:
Same comment as I made on https://github.com/dmranck/ticketutil/pull/52, please put link to Bugzilla API doc to show what format ticket_content will be returned in: http://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html#get-bug

Also, if you could add a quick note to this section about how the user accesses the ticket_content after running get_ticket_content(), i.e. returned_ticket_content = t.ticket_content, or something along those lines.

Reply:
changed